### DISCUSSION TITLE
Adding ability to change what user the shortcode belongs to on front end

### DISCUSSION THREAD

**Discussion by LeaveitT0uchless** [urania-dev/snapp#67] [Ideas]
In case you have some time :) it would be super if we could have the ability to change who can see/change the shortcode from the front end instead of doing it through the db like I'm doing now.

**Comment by urania-dev** [comment]
You mean change other people shortcodes? That's is already an admin power through ghosting users. (Sorry ive been caught up at work)

**Comment by LeaveitT0uchless** [comment]
I meant changing the ownership of a prefix(tag). Currently, I create an account for a user and set up all the shortcodes myself under that user and then I hand over the login details. It would be easier If I can have an admin that can do all the work from their own account and once the a new user onboards, those prefix(tags) can be moved over to be owned by that user which then allows them to see it and edit the short codes with that prefix(tag). 

This would also help when transferring ownership of that prefix(tag) to a new user when the old user should no longer have access to make changes to the shortcodes pertaining to that prefix(tag).

Hope that helps explain it better. No worries, I know this is a hobby for you and I'm just grateful that you take ideas and help fix things so quick. You're the best!

**Comment by urania-dev** [comment]
Please have a go with the new groups system, It could be more helpful for your scope

You can now create urls and share them with member in groups

**Comment by LeaveitT0uchless** [comment]
in my use case, the people part of that group should be able to edit them as users. Can that be implemented?

**Reply by urania-dev** [reply]
The groups info, members, or the snapp into the groups?

**Reply by LeaveitT0uchless** [reply]
Group members would have access to edit the snapp, but not members. I believe if this works, I can upgrade and then now need to depend on tags as prefix.

**Reply by urania-dev** [reply]
So group members can also edit other groups member snapp if in same group. The main difference would be this one.

If that s the case yeah. Its Just a zenstack policy update

**Reply by lit-carlos** [reply]
yes that's it

**Reply by urania-dev** [reply]
should be up in 007

**Comment by lit-carlos** [comment]
I think the group as prefix is not working any more. is there a way to continue the use case where someone can only make shortcodes as part of their group only and they are restricted to make only links for /testtag/shortcode only?

**Reply by urania-dev** [reply]
i set  a new URLS_VIA_GROUPS_ONLY=true for this use case, it will lock the creation of a new snapp if no groups is selected

**Reply by lit-carlos** [reply]
oh nice, is this implemented already?

**Reply by urania-dev** [reply]
building -013

**Reply by urania-dev** [reply]
up
